client/src/pages/private/SiteTable.js

```diff
@@ -286,48 +313,71 @@ export default ({ sites, viewOnly }) => {
         </Grid>
 
         <CheckPermissions roles={roles} permittedRoles={['ministry_of_health']}>
-          <Grid item xs={2} />
+          <Grid item xs={8} />
           <Grid className={classes.rootItem} item xs={2}>
-            <Button
-              onClick={() => {
-                setActiveModalForm('new-site');
-              }}
-              size='medium'
-              text='Create Site'
-              startIcon={<AddCircleOutlineIcon />}
-            />
+            <Box px={2} display='flex' justifyContent='end'>
+              <Button
+                onClick={openActionMenu}
+                endIcon={isActionMenuOpen ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+                aria-controls='simple-menu'
+                aria-haspopup='true'
+                text='Action'
+                variant='contained'
+                fullWidth={false}
+              />
+              <Menu
+                id='action-menu'
+                anchorEl={actionMenuAnchorEl}
+                open={Boolean(actionMenuAnchorEl)}
+                onClose={closeActionMenu}
+                getContentAnchorEl={null}
+                anchorOrigin={{ horizontal: 'left', vertical: 'bottom' }}
+                transformOrigin={{ horizontal: 'left', vertical: 'top' }}
+                PaperProps={{
+                  style: {
+                    minWidth: '220px',
+                  },
+                }}
+              >
+                <MenuItem onClick={openNewSiteModal} className={classes.menuItem}>
+                  Create new site
+                </MenuItem>
+                <FeatureFlag featureKey={flagKeys.FEATURE_PHASE_ALLOCATION}>
+                  <MenuItem onClick={openNewPhaseModal} className={classes.menuItem}>
+                    Create new phase
+                  </MenuItem>
+                </FeatureFlag>
+              </Menu>
+            </Box>
           </Grid>
         </CheckPermissions>
 
         {roles.includes('superuser') && <Grid item xs={8} />}
 
         {!viewOnly && (
           <>
-            <Grid item xs={6} />
-
-            <Grid container item xs={4}>
-              <Grid className={classes.rootItem} item xs={12}>
-                <CheckPermissions roles={roles} permittedRoles={['health_authority']}>
+            <CheckPermissions roles={roles} permittedRoles={['health_authority']}>
```

Comment:
This was rendering empty elements so I moved the `CheckPermissions` around to prevent that